DEVELOPMENT.md

```diff
@@ -68,6 +68,10 @@ Redis is an in-memory data store that we use as a database for "hot" data.
 
 You can use `tools/bin/mage dev:dbRedisCli` to enter a Redis-CLI shell.
 
+## Building the frontend
+
+You can use `tools/bin/mage js:build` to build the frontend.
```

Comment:
does `./mage` not work in `develop`? The rest of the documentation uses `./mage`

Reply:
https://github.com/TheThingsNetwork/lorawan-stack/pull/2568